pytest/test_io_meshes.py

```diff
@@ -290,3 +292,46 @@ def test_from_meshio_pygmsh(generate_pygmsh, tmp_path):
                                wavelength=True,
                                wavenumber=True)
 
+
+def test_load_gdf_compressed():
+    
+    body = cpt.HorizontalCylinder()
+    body_path = Path("temp_mesh.gdf")
+    write_GDF(str(body_path), body.mesh.vertices, body.mesh.faces, ulen=1, gravity=9.81, isx=0, isy=0)
+
+    body_from_gdf = load_GDF(str(body_path))
+    body_from_gdf_compressed = load_GDF_compressed(str(body_path))
+    
+    np.testing.assert_allclose(
+        body_from_gdf.vertices[body_from_gdf.faces], 
+        body_from_gdf_compressed.vertices[body_from_gdf_compressed.faces]
+        )
+    
+
+    np.testing.assert_allclose(
+        body_from_gdf.faces_areas, 
+        body_from_gdf_compressed.faces_areas
+        )
+    
+    np.testing.assert_allclose(
+        body_from_gdf.faces_normals, 
+        body_from_gdf_compressed.faces_normals
+        )
+    
+    np.testing.assert_allclose(
+        body_from_gdf.faces_centers, 
+        body_from_gdf_compressed.faces_centers
+        )
+    
+    np.testing.assert_allclose(
+        body_from_gdf.faces_radiuses, 
+        body_from_gdf_compressed.faces_radiuses
+        )
+    
+    np.testing.assert_allclose(
+        body_from_gdf.volume, 
+        body_from_gdf_compressed.volume
+        )
+    
+    body_path.unlink()
```

Comment:
FYI, there a feature in Pytest to create temporary directories, so you don't have to cleanup afterwards.
https://docs.pytest.org/en/6.2.x/tmpdir.html